brave/browser-laptop
# Refresh Swedish language strings

Auditors: @clifton

Test Plan:

  * Change language to Swedish, restart
  * Ensure new strings are displayed

Fixes: #11298

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
